widgetssdk/src/main/java/com/glia/widgets/call/CallView.kt

```diff
@@ -974,9 +974,9 @@ internal class CallView(
                 callState.callStatus.formattedOperatorName,
                 ""
             )
-            if (callState.companyName != null) { // TODO FIXME Make CompanyName configurable from remote strings
+            if (callState.companyName != null) {
                 companyNameView.text = callState.companyName
-                companyNameView.hint = stringProvider.getRemoteString(R.string.general_company_name)
+                companyNameView.hint = callState.companyName
```

Comment:
@Pelkar  It seems to me that we need a separate string that will be used as an accessibility label. Basically, it should be a translation for the "Company name" wording.

So, one string for the name itself and another one just as a literal translation for "Company name".

As far as I understand, currently, we have only one string for the name. And at the moment, the name of the company serves as an accessibility label. Not critical, but could be improved.

Reply:
We discussed this with @gersonnoboa before he left for vacation and the decision was to have the accessibility text just be the company name itself. For now, this should remain the case, until further notice. We can always improve on this later on if the need arises.